addon/utils/dom-helpers.ts

```diff
@@ -413,6 +413,31 @@ function isVisibleElement(element: HTMLElement): boolean {
   return !!(element.offsetWidth || element.offsetHeight || element.getClientRects().length);
 }
 
+/**
+ * Utility to get the editor element in a type-safe way
+ * This avoids having to nullcheck everywhere where a null editor would be an error anyway.
+ */
+function getEditorElement(): Element {
```

Comment:
if this is only used in tests (guessing here), this file is probably not the right location

Reply:
well so far it is, but I thought it could also be useful elsewhere

Reply:
I'm a bit wary, because typically it would be best to retrieve the element using rawEditor.rootNode . the selection is a bit brittle imo.

Reply:
ah yeah now that I look at it again its not ideal
I'll make a testutils file